Formula/linkerd2.rb

```diff
@@ -0,0 +1,43 @@
+class Linkerd2 < Formula
+  desc "Linkerd configuration command-line utility"
+  homepage "https://linkerd.io"
+
+  url "https://github.com/linkerd/linkerd2.git",
+    :tag      => "stable-2.2.0",
+    :revision => "1cde1f083c5f245170f75e1974f3b20f7fa34d9c"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CI_FORCE_CLEAN"] = "1"
+
+    srcpath = buildpath/"src/github.com/linkerd/linkerd2"
+    srcpath.install buildpath.children - [buildpath/".brew_home"]
+
+    cd srcpath do
+      system "bin/build-cli-bin"
+      bin.install "target/cli/darwin/linkerd"
```

Comment:
No doc, no man page?

Reply:
There is a documentation site: https://linkerd.io/2/overview/. Not sure how to put reference for that.

linkerd is a go binary and "help" is built into it. Example is below, not sure if it would suffice

```
 $ > linkerd help
linkerd manages the Linkerd service mesh.

Usage:
  linkerd [command]

Available Commands:
  check       Check the Linkerd installation for potential problems
  completion  Shell completion
  dashboard   Open the Linkerd dashboard in a web browser
  endpoints   Introspect Linkerd's service discovery state
  get         Display one or many mesh resources
  help        Help about any command
  inject      Add the Linkerd proxy to a Kubernetes config
  install     Output Kubernetes configs to install Linkerd
  install-sp  Output Kubernetes configs to install Linkerd Service Profiles
  logs        Tail logs from containers in the Linkerd control plane
  profile     Output service profile config for Kubernetes
  routes      Display route stats
  stat        Display traffic stats about one or many resources
  tap         Listen to a traffic stream
  top         Display sorted information about live traffic
  uninject    Remove the Linkerd proxy from a Kubernetes config
  version     Print the client and server version information
```